pulp_container/app/viewsets.py

```diff
@@ -634,7 +634,14 @@ class ContainerDistributionViewSet(BaseDistributionViewSet):
                 "effect": "allow",
                 "condition": [
                     "has_model_or_obj_perms:container.view_containerdistribution",
-                    # "has_namespace_obj_perms:container.view_containernamespace",
+                ],
+            },
+            {
+                "action": [],
```

Comment:
I think there should be an action. Even if it is not defined on _that_ viewset.

Reply:
actually this was on purpose. If i will keep 'get' for example i will be able to still see the distribution endpoint with pull permissions. The goal was to separate view and pull perms. Meaning that if a user can pull content of a distribution it does not mean that he can do GET on pulp api distribution endpoint and see data.

Reply:
> I think there should be an action. Even if it is not defined on _that_ viewset.

i don't think so https://rsinger86.github.io/drf-access-policy/statement_elements/#action that;s why i left it empty

Reply:
How is this policy then used anywhere?

Reply:
we are checking on users perms during push/pull operations

Reply:
I believe `user.has_perm` is already one level beneath the access policy (It is the technology used by the policies rules.).